src/test/java/uk/gov/justice/probation/courtcaseservice/controller/model/CaseCommentRequestTest.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.probation.courtcaseservice.controller.model;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.probation.courtcaseservice.jpa.entity.CaseCommentEntity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CaseCommentRequestTest {
+
+    @Test
+    void asEntity() {
+        var caseCommentRequest = CaseCommentRequest.builder()
+            .comment("awaiting PSR")
+            .caseId("test-cae-id")
+            .build();
+
+        assertThat(caseCommentRequest.asEntity())
+            .isEqualTo(CaseCommentEntity.builder()
+                .comment(caseCommentRequest.getComment())
```

Comment:
I'm not sure how I feel about using the getters from the `caseCommentRequest` here 🤔 It reads in a sort of circular manner because it refers back to the values of the test subject. It's also identical to the actual code in the `asEntity` method which doesn't feel right, as a reader it looks like it could have been copy-pasted.  

There's also an inconsequential typo on line 14 which the test didn't pick up. It doesn't really matter, it's still exercising the code, but it just doesn't look quite right to me.